bin/collator/src/parachain/service.rs

```diff
@@ -606,6 +602,12 @@ pub struct StartupConfiguration {
     pub proposer_soft_deadline_percent: u8,
 }
 
+/// To add additional config to start_xyz_node functions
+pub struct AdditionalConfig {
+    /// EVM tracing configuration
+    pub evm_tracing_config: Option<crate::evm_tracing_types::EvmTracingConfig>,
```

Comment:
Might as well import the type above.

Also, should this always be included or should a feature flag be used instead?

Reply:
I think Option makes it more idiomatic because if we use feature flags we will have to pass empty struct in function to make the code compatible in many places.

Reply:
If only this parameter is kept, true.
However, I don't think it should be empty.

See my other comment in this review.

Reply:
Sure, let's use a feature flag for `evm_tracing_config`